### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by johnpenxa**

### CODE REVIEW

**Comment by johnpenxa** [comment]
mypull

File: fall_2017/JohnPenxa/swagger/swagger.yaml [hunk 1]
```diff
@@ -0,0 +1,326 @@
...
+      tags:
+        - users
+      summary: Update user information
+      description: update user info
+      operationId: updateUserInfo
+      consumes:
+        - application/json
+      produces:
+        - application/json
+      parameters:
```
Comment (toddysm) [inline fall_2017/JohnPenxa/swagger/swagger.yaml]: Missing {id} parameter in the path

File: fall_2017/JohnPenxa/swagger/swagger.yaml [hunk 2]
```diff
@@ -0,0 +1,326 @@
...
+    delete:
+      tags:
+        - users
+      summary: Delete user from system
+      description: delete user ID
+      operationId: deleteUser
+      produces:
+        - application/json
+      parameters:
+        - name: api_key
```
Comment (toddysm) [inline fall_2017/JohnPenxa/swagger/swagger.yaml]: we don't need the API key

File: fall_2017/JohnPenxa/swagger/swagger.yaml [hunk 3]
```diff
@@ -0,0 +1,326 @@
...
+          description: new task to add
+          required: false
+          schema:
+            $ref: '#/definitions/Task'
+      responses:
+        '201':
+          description: task created
+        '400':
+          description: invalid input
+  '/users/{id}/tasks/{id}':
```
Comment (toddysm) [inline fall_2017/JohnPenxa/swagger/swagger.yaml]: You need different names for the parameters

File: fall_2017/JohnPenxa/swagger/swagger.yaml [hunk 4]
```diff
@@ -0,0 +1,326 @@
...
+    get:
+      tags:
+        - tasks
+      summary: Look up a task by ID
+      description: find a task
+      operationId: taskIdSearch
+      produces:
+        - application/json
+      parameters:
+        - name: id
```
Comment (toddysm) [inline fall_2017/JohnPenxa/swagger/swagger.yaml]: You need to update with task ID parameter

File: fall_2017/JohnPenxa/swagger/swagger.yaml [hunk 5]
```diff
@@ -0,0 +1,326 @@
...
+      tags:
+        - tasks
+      summary: Update a user's task
+      description: update task
+      operationId: updateTask
+      consumes:
+        - application/json
+      produces:
+        - application/json
+      parameters:
```
Comment (toddysm) [inline fall_2017/JohnPenxa/swagger/swagger.yaml]: Update with path parameters (missing user ID and task ID)

File: fall_2017/JohnPenxa/swagger/swagger.yaml [hunk 6]
```diff
@@ -0,0 +1,326 @@
...
+          description: task ID not found
+    delete:
+      tags:
+        - tasks
+      summary: Delete a user's task
+      description: delete task
+      operationId: deleteTask
+      produces:
+        - application/json
+      parameters:
```
Comment (toddysm) [inline fall_2017/JohnPenxa/swagger/swagger.yaml]: Remove api_key and add task ID

**Pull request merged**